stable/datadog/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: datadog
-version: 1.5.2
+version: 1.5.3
```

Comment:
it's more of a minor change than a bugfix, you can bump the minor version instead.